src/pages/VideoPage/VideoPage.component.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useContext, useEffect } from 'react';
+import { useParams } from 'react-router';
+import { Link } from 'react-router-dom';
+import { DataContext } from '../../providers/Data';
+import './VideoPage.styles.css';
+// import { videoData } from '../../utils/dataMock';
+
+function VideoPage() {
+  const [loading, setLoading] = useState(false);
+  const [videoData, setVideoData] = useState([]);
+  const data = useContext(DataContext);
+  const relatedVideos = JSON.parse(JSON.stringify(videoData)).slice(1);
+
+  const { videoId } = useParams();
+
+  useEffect(() => {
+    setLoading(true);
+    if (data) {
+      setVideoData(data);
+    }
+    setLoading(false);
+  }, [data]);
+
+  console.log(videoData);
+
+  return (
+    <section className="video-view">
+      <p>Video Page Here</p>
+      {!loading && (
+        <iframe
+          className="i-frame"
+          allowFullScreen
+          frameBorder="0"
+          title="Título"
+          src={`https://www.youtube.com/embed/${videoId}?controls=0`}
+          allow="accelerometer; encrypted-media; gyroscope; picture-in-picture"
+        />
```

Comment:
This could be extracted as another component and used later again at line 44, the difference would be the provided parameters